### ISSUE TITLE
100% dry wax never hardening

### ISSUE THREAD

**Issue by baronjutter** [ruggbean/More-Candles#1]
### Mod Version

1.0.3

### Modded

Required food mods

### SP/MP

Multiplayer

### Description

```shell
I'm not sure if its a bug or just bad/buggy feedback on the tooltips, but I have a basement full of shelves of 100% dry saucepans of beeswax.  All still liquid. If I take one off and put it back on, it resets the progress to about 50-60%.  

Also the initial tooltip for the wax on the shelf said it would only take a couple days to dry, mine have been sitting for a week+ and only very slowly creeping up in dryness.
```

### How to Reproduce

```shell
Boil up a bunch of wax, dump into saucepan, put on shelf.  Repeat a few times until you have a shelf full of saucepans of various states of drying.  Weeks go by, percentage slowly creeps up on all of them until some reach 100%.  They never turn into dry wax.
```

**Comment by ruggbean** [comment]
Hello. I am currently working on a fix for what I believe to be a problem with the wax as a whole, which is possibly the state of the dried wax conflicting with the drying stage. 

Would you be able to attach a copy of your client-main logs (and client-crash, if this error causes your game to crash)

**Comment by ruggbean** [comment]
Hello. I have released 1.1.1 to hopefully address the issues with the drying wax. I will be closing out this issue, however, please come back if you continue having further issues.
